### PR TITLE
config/shell: add some available shells

### PR DESCRIPTION
**Pull request by pltrz**

### CODE REVIEW

File: src/config/shell.md [hunk 1]
```diff
@@ -32,15 +32,18 @@ shells.
 
 - bash
 - dash
+- dreampie
```
Comment (Vaelatern) [inline src/config/shell.md]: No

File: src/config/shell.md [hunk 2]
```diff
@@ -32,15 +32,18 @@ shells.
...
 - elvish
 - es
 - fish-shell
 - heirloom-sh
+- ion
```
Comment (Vaelatern) [inline src/config/shell.md]: Ok

File: src/config/shell.md [hunk 3]
```diff
@@ -32,15 +32,18 @@ shells.
...
 - ksh
-- mksh
 - loksh
+- mksh
 - posh
 - rc
 - tcsh
+- xonsh
```
Comment (Vaelatern) [inline src/config/shell.md]: I guess

File: src/config/shell.md (hunk 1 above)
Comment (Vaelatern) [reply src/config/shell.md]: This is not an available shell.

Comment (pltrz) [reply src/config/shell.md]: My mistake. I Just went and installed all the shells I could find and looked at which ones added entries to /etc/shells. This one does not though.

Comment (pltrz) [reply src/config/shell.md]: Removed it in the latest push

**Pull request merged**